src/components/apps/listing/Listing.js

```diff
@@ -109,28 +97,61 @@ function Listing({
         isFetching: appInCategoryStatus,
         error: appsInCategoryError,
     } = useQuery({
-        queryKey: appsInCategoryKey,
+        queryKey: [
+            APPS_IN_CATEGORY_QUERY_KEY,
```

Comment:
I left a note in my commit message, but I switched to using this method of query keys instead of the state and effect combo because I found during testing that having the key switch to null and enabled set to false doesn't seem to clear out the error, in this case `appsInCategoryError`.  So, for example, I could select `Browse All Apps` and see the listing, select any other category and see the `Sign In` page, switch back to `Browse All Apps` and I would still see the `Sign In` page instead of the listing.

I also found this easier to read because I didn't have to jump up and down between state and effect functions to see when these queries were supposed to be active.